IdentityCore/src/requests/MSIDInteractiveAuthorizationCodeRequest.h

```diff
@@ -36,6 +37,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, readonly) MSIDInteractiveTokenRequestParameters *requestParameters;
 @property (nonatomic, readonly) MSIDOauth2Factory *oauthFactory;
+@property (nonatomic, copy) MSIDExternalDecidePolicyForBrowserActionBlock externalDecidePolicyForBrowserAction;
```

Comment:
Double check for retain cycles since you're copying the block

Reply:
Updated and removed all references to `self` when using this block.